src/compiler.js

```diff
@@ -121,6 +121,10 @@ var Compiler = Object.extend({
         return 't_' + this.lastId;
     },
 
+    _templateName: function() {
+        return this.templateName == null? "undefined" : '"'+this.templateName.replace(/"/g, '\\"')+'"';
```

Comment:
Ok. It doesn't hurt. Eventually we could have a top-level method in `compiler.js` that does this quote escaping, but this is fine for now.